website/content/hooks/useContract.mdx

```diff
@@ -48,10 +51,13 @@ const abi = [
     },
 ];
 export default function Component() {
-    const { contract } = useContract({ abi, address: testAddress })
+    const provider = publicProvider()
 
+    const { contract } = useContract({ abi: abi, address: testAddress, provider: provider(devnet) })
```

Comment:
```suggestion
    const { contract } = useContract({ abi: abi, address: testAddress })
```

While we provide a way to override the current provider, by default (and in 99% of the cases) developers will want to use the current provider. So let's keep the example simple.